internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -250,17 +250,28 @@ func convertEFSPermissions(input map[string]manifest.Volume) ([]*template.EFSPer
 	}
 	output := []*template.EFSPermission{}
```

Comment:
```suggestion
	var output []*template.EFSPermission
```

Reply:
^ I am so proud, @kohidave I have finally won the war

Reply:
You may have won the battle. I waive no white flag.

Reply:
don't take it personally lol its just you won't get `[]*template. EFSPermission{} == nil` to be true.